docs/site-example/i18n/en.po

```diff
@@ -17,17 +17,18 @@ msgstr ""
 
 msgid "gluon-config-mode:pubkey"
 msgstr ""
-"This is your Freifunk node's public key. The node won't be able to "
-"connect to the mesh VPN until the key has been registered on the Freifunk "
-"Duckburg servers. "
+"This is the public key of your Freifunk node. "
+"Your node may either connect to nearby nodes through wifi "
+"or to gateway servers via Mesh-VPN. "
+"Last option is only possible after the key has been added to the Server. "
```

Comment:
Wie wäre es mit: Connecting via Mesh-VPN requires the key to be enabled by your community.